index.html

```diff
@@ -1,13 +1,152 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Poppins"
+    />
+    <link rel="stylesheet" href="/src/styles/styles.css" />
     <title>Practice 3 - Ramiro</title>
   </head>
+
   <body>
-    <div id="app"></div>
     <script type="module" src="/src/main.ts"></script>
```

Comment:
It is suggested to always import scripts at the end, in the desired execution order, if you have multiple imports.